autoload/asyncomplete.vim

```diff
@@ -447,7 +447,7 @@ function! s:python_refresh_completions(ctx) abort
         endif
     endfor
 
-    call s:core_complete(a:ctx, l:startcol, l:filtered_matches, s:matches)
+    call timer_start(g:asyncomplete_popup_delay, {timer->s:core_complete(a:ctx, l:startcol, l:filtered_matches, s:matches)})
```

Comment:
If `0` can we not directly call `s:core_complete` without `timer_start`?

Reply:
Hm I don't know the best way to do this; if we do `timer_start` with `0` ms, it just happens immediately anyway. Is it worth adding an extra conditional?

Reply:
I'm worried about this https://github.com/vim/vim/issues/1713#issuecomment-455795951.

Reply:
Good point -- fixed.